tools/clang/test/LitDXILValidation/load-store-validation.ll

```diff
@@ -1,3 +1,4 @@
+; REQUIRES: dxil-1-9
```

Comment:
I think we should start a convention where we create sub-folders under `LitDXILValidation` for specific validator version requirements.  All new tests would go into the latest folder, and we would have a lit.local.cfg that implements this requirement, so that none of the tests are required to individually state this requirement.

Reply:
https://github.com/microsoft/DirectXShaderCompiler/issues/7413